src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -733,14 +752,24 @@ private void refreshBeansList() throws IOException {
                     ObjectName name = new ObjectName(scope);
                     this.beans.addAll(connection.queryNames(name));
                 }
-            } catch (Exception e) {
+                if (instanceTelemetryBean != null) {
+                    instanceTelemetryBean.setDomainsQueried(beanScopes.size());
```

Comment:
Does one scope correspond to one domain?
We may want to unlink these from each other, presumably `getBeansScopes` could return scopes that are more specific than a domain, even if today all it does is return the domain.

Or rename `getBeansScopes` to `getBeansDomains` if that is indeed all it does.